recipes-kernel/linux/files/patches-6.1/0082-dts-iot2050-Support-IOT2050-SM-variant.patch

```diff
@@ -0,0 +1,400 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Tue, 19 Sep 2023 10:51:24 +0800
+Subject: [PATCH] dts: iot2050: Support IOT2050-SM variant
+
+Main differences between the new variant and Advanced PG2:
+
+1. Arduino interface is removed. Instead, an new ASIC is added for
+   communicating with PLC 1200 signal modules.
+2. USB 3.0 type A connector is removed, only USB 2.0 type A connector is
+   avaiable.
+3. DP interface is tailored down. Instead, to communicate with the
+   PLC 1200 signal modules, a USB 3.0 type B connector is added but the
+   signal is not USB.
+4. DDR size is increased to 4 GB.
+5. Two sensors are added, one tilt sensor and one light sensor.
+
+Co-developed-by: Chao Zeng <chao.zeng@siemens.com>
+Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
+Co-developed-by: Li Hua Qian <huaqian.li@siemens.com>
+Signed-off-by: Li Hua Qian <huaqian.li@siemens.com>
+Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
+---
+ .../devicetree/bindings/arm/ti/k3.yaml        |   1 +
+ arch/arm64/boot/dts/ti/Makefile               |   1 +
+ .../dts/ti/k3-am6548-iot2050-advanced-sm.dts  | 341 ++++++++++++++++++
+ 3 files changed, 343 insertions(+)
+ create mode 100644 arch/arm64/boot/dts/ti/k3-am6548-iot2050-advanced-sm.dts
+
+diff --git a/Documentation/devicetree/bindings/arm/ti/k3.yaml b/Documentation/devicetree/bindings/arm/ti/k3.yaml
+index d16231bdee6e..130a4f9e12cc 100644
+--- a/Documentation/devicetree/bindings/arm/ti/k3.yaml
++++ b/Documentation/devicetree/bindings/arm/ti/k3.yaml
+@@ -44,6 +44,7 @@ properties:
+               - siemens,iot2050-advanced
+               - siemens,iot2050-advanced-m2
+               - siemens,iot2050-advanced-pg2
++              - siemens,iot2050-advanced-sm
+               - siemens,iot2050-basic
+               - siemens,iot2050-basic-pg2
+               - ti,am654-evm
```

Comment:
This must be a separate patch for upstream.

Reply:
Fixed in #509